Mail/Views/Thread/ThreadView.swift

```diff
@@ -86,6 +86,8 @@ struct ThreadView: View {
                             ExternalRecipientView(externalTagSate: externalTag, isDraft: false)
                         }
                     }
+
+                    MessageFolderCell(title: thread.searchFolderName, inThreadHeader: true)
```

Comment:
If possible, MessageFolderCell (which is actually a tag) should be displayed next to the external tag. Maybe we can use a WrappingHStack